f5_openstack_agent/lbaasv2/drivers/bigip/test/test_exceptions.py

```diff
@@ -0,0 +1,521 @@
+
+
+def test_SNATDeleteException():
+    with pytest.raises(exceptions.SNATDeleteException):
+        raise getattr(exceptions, 'SNATDeleteException')()
+
+
+def test_SystemCreationException():
+    with pytest.raises(exceptions.SystemCreationException):
+        raise getattr(exceptions, 'SystemCreationException')()
+
+
+def test_SystemQueryException():
+    with pytest.raises(exceptions.SystemQueryException):
+        raise getattr(exceptions, 'SystemQueryException')()
+
+
+def test_SystemUpdateException():
+    with pytest.raises(exceptions.SystemUpdateException):
+        raise getattr(exceptions, 'SystemUpdateException')()
+
+
+def test_SystemDeleteException():
+    with pytest.raises(exceptions.SystemDeleteException):
+        raise getattr(exceptions, 'SystemDeleteException')()
+
+
+def test_VirtualServerCreationException():
+    with pytest.raises(exceptions.VirtualServerCreationException):
+        raise getattr(exceptions, 'VirtualServerCreationException')()
+
+
+def test_VirtualServerQueryException():
+    with pytest.raises(exceptions.VirtualServerQueryException):
+        raise getattr(exceptions, 'VirtualServerQueryException')()
+
+
+def test_VirtualServerUpdateException():
+    with pytest.raises(exceptions.VirtualServerUpdateException):
+        raise getattr(exceptions, 'VirtualServerUpdateException')()
+
+
+def test_VirtualServerDeleteException():
+    with pytest.raises(exceptions.VirtualServerDeleteException):
+        raise getattr(exceptions, 'VirtualServerDeleteException')()
+
+
+def test_VLANCreationException():
+    with pytest.raises(exceptions.VLANCreationException):
+        raise getattr(exceptions, 'VLANCreationException')()
+
+
+def test_VLANQueryException():
+    with pytest.raises(exceptions.VLANQueryException):
+        raise getattr(exceptions, 'VLANQueryException')()
+
+
+def test_VLANUpdateException():
+    with pytest.raises(exceptions.VLANUpdateException):
+        raise getattr(exceptions, 'VLANUpdateException')()
+
+
+def test_VLANDeleteException():
+    with pytest.raises(exceptions.VLANDeleteException):
+        raise getattr(exceptions, 'VLANDeleteException')()
+
+
+def test_VXLANCreationException():
+    with pytest.raises(exceptions.VXLANCreationException):
+        raise getattr(exceptions, 'VXLANCreationException')()
+
+
+def test_VXLANQueryException():
+    with pytest.raises(exceptions.VXLANQueryException):
+        raise getattr(exceptions, 'VXLANQueryException')()
+
+
+def test_VXLANUpdateException():
+    with pytest.raises(exceptions.VXLANUpdateException):
+        raise getattr(exceptions, 'VXLANUpdateException')()
+
+
+def test_VXLANDeleteException():
+    with pytest.raises(exceptions.VXLANDeleteException):
+        raise getattr(exceptions, 'VXLANDeleteException')()
+
+
+def test_BigIPNotLicensedForVcmp():
+    with pytest.raises(exceptions.BigIPNotLicensedForVcmp):
+        raise getattr(exceptions, 'BigIPNotLicensedForVcmp')()
+
+
```

Comment:
The above tests just seem like coverage extenders, let's at least move them to the end of the module.